### ISSUE TITLE
Platform update notification is not working on Mulitsites

### ISSUE THREAD

**Issue by emmanoctaviano** [buddyboss/buddyboss-platform#597]
**Describe the bug**
Customers who are using multisite could not update their BuddyBoss Platform plugin automatically.

**To Reproduce**
1. Set up multisite
2. Install the previous versions of BuddyBoss Theme/Platform
3. Ensure license keys are connected

**Expected behavior**
The BuddyBoss Theme has available updates but the BuddyBoss Platform don't have notifications

**Screenshots**
https://drive.google.com/open?id=1Pr-3TSddHRWDVzB4nppNgdM_heuWqNHo

**Support ticket links**
https://secure.helpscout.net/conversation/1111922363/1469/

**Comment by janakkaneriya007** [comment]
@emmanoctaviano 

I have tried by installing the older version of the BuddyBoss Platform plugin but I was able to get the update notification for the newer version.

Also, what you are showing in video is two different things, You are adding license for the BuddyBoss theme and checking the update for the BuddyBoss Platform plugin which is not related.

Might be possible that you are already up to date with the Platform plugin version and it's not showing the update notification for that.

If the issue is still reproducible let me know the exact steps to reproduce it.
